languagetool-language-modules/fr/src/main/resources/org/languagetool/rules/fr/grammar.xml

```diff
@@ -20156,6 +20156,20 @@ though, and has since been slightly modified)-->
       <example correction="De toute façon"><marker>De toutes façons</marker></example>
       <example><marker>De toute façon</marker></example>
     </rule>
+      <rule id="A_TOUTE_A_L_HEURE" name="à tout à l'heure">
```

Comment:
Always add `default="temp_off"` here, to test the rule before enabling it in production, even if you think it is completely safe because sometimes it is not.

Reply:
1) Jaume is right, I forgot to tell you about this.

2) It might be worth it making this a `rulegroup`. This rule would be the first `<rule>`, and then you could make a second `<rule>` for "tout(e) à l'heure" (without à).

_Je vais le lui dire tout**e** à l'heure._

(There might be false positives, if "toute" just happens to appear before phrases like "à l'heure qu'il est" or "à l'heure actuelle")

3) I've seen that you've put this at the end of the `category`. Could you put new rules on top, instead? Also, I've seen that you've put this in the category `CAT_REGIONALISMES`, I think it's rather `CAT_GRAMMAIRE`.